lab-saul/lib/print-files.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+
+const fs = require('fs');
+
+
+const that = module.exports = function (paths, callback){
+  let hexArray = [];
+  fs.readFile(paths[0], function(err,data){
+    if(err) throw err;
+
+    let firstBuffer = data.toString('utf8', 0);
+    hexArray.push(firstBuffer);
+
+    fs.readFile(paths[1], function(err,data){
+      if (err) throw err;
+
+      let secondBuffer = data.toString('utf8', 0);
+      hexArray.push(secondBuffer);
+
+
+      fs.readFile(paths[2], function(err,data){
+        if(err) return callback(err);
```

Comment:
This pattern is spot on! Why not apply this same pattern to the other two instances in this file where you need to throw an error if needed (on lines 10 and 16)?